### PR TITLE
Handle version command first

### PR DESCRIPTION
**Pull request by trushton**
resolves [exosphere-sdk#248](https://github.com/Originate/exosphere-sdk/issues/248)

The `require` statement that pulls in the exo commands seems to be causing some lag. In the case that we run `exo version`, we want that to return quickly, so the code for handling that command was lifted to the top of the file.

@kevgo

### CODE REVIEW

**Comment by trushton** [comment]
It also seems that the last time the sdk was updated, the version number of `exorelay` did not receive the update. The tests are looking to install version `0.21.5` but the latest available version of `exorelay` on npm is `0.21.4`

**Review by kevgo — COMMENTED** [review]
Thanks for looking into this! Which require exactly is the slow one? They should all be pretty fast, no?

**Comment by kevgo** [comment]
If requiring all the commands is slow, maybe we should only require the one that is actually executed?

**Comment by kevgo** [comment]
After thinking more about this, there is no need to statically require all commands all the time. Let's only require the command that is being executed.

**Comment by trushton** [comment]
@kevgo This works much better now

File: cli/src/cli.ls [hunk 1]
```diff
@@ -20,24 +11,23 @@ require! {
...
+  run: "../../exo-run"
+  setup: "../../exo-setup"
+  sync: "../../exo-sync"
+  test: "../../exo-test"
 
 command-name = process.argv[2]
 full-command-name = complete-command-name command-name
 if command-name is \version
-  console.log "Exosphere version #{pkg.version}"
+  return console.log "Exosphere version #{pkg.version}"
```
Comment (kevgo) [inline cli/src/cli.ls:30]: If you add a `return` here, you can remove the `else` on the next line.

**Review by kevgo — APPROVED** [review]
Nice! The tests are red now. Feel free to ship when they are green!

**Pull request closed without merging**